### PR TITLE
Improve utils/constructors.py

### PR DESCRIPTION
**Pull request by Muxas**

### CODE REVIEW

File: wrappers/python/nntile/utils/constructors.py [hunk 1]
```diff
@@ -39,38 +40,113 @@
...
-    return from_array(np.zeros(shape, dtype=np_dtype))
+def zeros(
+    shape: Sequence[int],
+    basetile_shape: Sequence[int] | None = None,
+    dtype: Tensor = Tensor_fp32,
+    mpi_distr: Sequence[int] | None = None,
+    next_tag: int = 0,
+):
+    A_value = empty(shape, basetile_shape, dtype, mpi_distr, next_tag)
+    clear_async(A_value)
```
Comment (svtdanny) [inline wrappers/python/nntile/utils/constructors.py:96]: Do we have strong guarantee about clear_async doing actual filling with zeros?
Isn't it better to use explicit fill_async(0, A_value)

Comment (Muxas) [reply wrappers/python/nntile/utils/constructors.py:96]: 'clear_async' actually calls memset(0) under the hood. It sets every byte to 0. So, there is a strong guarantee all integer, bool and floating point values are zeros of proper type

File: wrappers/python/nntile/utils/constructors.py [hunk 2]
```diff
@@ -39,38 +40,113 @@
...
+    mpi_distr: Sequence[int] | None = None,
+    next_tag: int = 0,
+):
+    A_value = empty(shape, basetile_shape, dtype, mpi_distr, next_tag)
+    fill_async(fill_value, A_value)
+    return A_value
+
+
+def full_like(
+    A: np.ndarray,
```
Comment (svtdanny) [inline wrappers/python/nntile/utils/constructors.py:124]: Should we pass nntile tensor here? It will be more convenient

Comment (Muxas) [reply wrappers/python/nntile/utils/constructors.py:124]: This would be a nice feature, indeed. I will add it in this PR.

Comment (svtdanny) [reply wrappers/python/nntile/utils/constructors.py:124]: just to be sure we think about the same things, I meant to do:
A: nntile.Tensor,

Same notice for zeros_like

**Review by svtdanny — APPROVED** [review]

**Pull request merged**